packages/remix/src/client/browserTracingIntegration.ts

```diff
@@ -0,0 +1,41 @@
+import { browserTracingIntegration as originalBrowserTracingIntegration } from '@sentry/react';
+import type { Integration } from '@sentry/types';
+import { setGlobals, startPageloadSpan } from './performance';
+import type { RemixBrowserTracingIntegrationOptions } from './performance';
+/**
+ * Creates a browser tracing integration for Remix applications.
+ * This integration will create pageload and navigation spans.
+ */
+export function browserTracingIntegration(options: RemixBrowserTracingIntegrationOptions): Integration {
+  if (options.startTransactionOnPageLoad === undefined) {
+    options.startTransactionOnPageLoad = true;
+  }
+
+  if (options.startTransactionOnLocationChange === undefined) {
+    options.startTransactionOnLocationChange = true;
+  }
```

Comment:
do we need this? Instead we should check `instrumentPageLoad` and `instrumentNavigation` here!